test/TestProjects/MgmtPropertyChooser/Generated/Models/CloudError.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using Azure.ResourceManager.Resources.Models;
+
+namespace MgmtPropertyChooser.Models
+{
+    /// <summary> An error response for a resource management request. </summary>
+    internal partial class CloudError
```

Comment:
I assume this is also waiting for the PropertyReferenceType attribute on ErrorResponse?

Reply:
Yes, now I generate the Models based on local changes of Azure.ResourceManager.